src/screens/Episode/helpers/getTime.ts

```diff
@@ -0,0 +1,17 @@
+import * as dateFns from 'date-fns'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+
+const getTime = (amntOfSeconds: number): string => {
+  const HHmmss = getFormattedDate(
+    String(dateFns.addSeconds(new Date(0), amntOfSeconds)),
```

Comment:
What does it mean `0`? Make constant for it, please
```suggestion
    String(dateFns.addSeconds(new Date(0), seconds)),
```